pkg/kubelet/kubelet_node_status.go

```diff
@@ -350,6 +350,9 @@ func (kl *Kubelet) updateNodeStatus() error {
 	glog.V(5).Infof("Updating node status")
 	for i := 0; i < nodeStatusUpdateRetry; i++ {
 		if err := kl.tryUpdateNodeStatus(i); err != nil {
+			if i > 0 && kl.onRepeatedHeartbeatFailure != nil {
+				kl.onRepeatedHeartbeatFailure()
```

Comment:
Do we want to set onRepeatedHeartbeatFailure to nil or something? (once we invoke the method)

Reply:
No, that would mean the kubelet would hit the same issue if the network condition was encountered twice during a single process lifetime

Reply:
Ack thanks!